src/applications/mhv/landing-page/components/NavCard.jsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import recordEvent from '~/platform/monitoring/record-event';
+import { getHostname } from '../utilities';
 
 const NavCard = ({ icon = null, title, links }) => {
-  const listItems = links.map(l => (
-    <li className="mhv-c-navlistitem" key={l.key || l.href}>
-      <a className="mhv-c-navlink" href={l.href} aria-label={l.ariaLabel}>
-        {l.text}
+  const listItems = links.map(({ ariaLabel, href, text }) => (
+    <li className="mhv-c-navlistitem" key={href}>
+      <a
+        className="mhv-c-navlink"
+        href={href}
+        aria-label={ariaLabel}
+        onClick={() =>
+          recordEvent({
+            event: 'nav-linkslist',
+            'links-list-title': String(text) === text ? text : 'Inbox', // hack to handle 'the dot'
+            'links-list-group': title,
+            'links-list-hostname': getHostname(href),
```

Comment:
When I grep for `recordEvent` elsewhere in the code, I don't see other apps setting any hostname parameters. I'd expect GA to be capturing the URL, and thus the hostname, directly, so I'm wondering if this and the `getHostname` aren't needed?

Reply:
```suggestion
```

Reply:
These changes are now in thanks

Reply:
The goal was to start collecting data on "Destination domains" we were pointing users to, as per [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66723#issuecomment-1789106226) in the origin ticket.